source/api_documentation/signal_api/_create_call_outcome.rst

```diff
@@ -75,8 +75,8 @@
         "advertiser_campaign_id_from_network": "2",
         "network_id": "3"
       },
-      "call_outcome": {
-        "outcome_name": "sale",
+      "signal": {
+        "name": "sale",
         "partner_unique_id": "1",
```

Comment:
Quick question. Will the file upload csv template now use the column header "name" instead of "outcome_name" as well?